src/commands/overrides/update.ts

```diff
@@ -0,0 +1,93 @@
+    ]
+    static flags = {
+        ...Base.flags,
+        feature: Flags.string({
+            description: 'The feature to update an override for'
+        }),
+        environment: Flags.string({
+            description: 'The environment to update an override for'
+        }),
+        variation: Flags.string({
+            description: 'The variation to update an override for'
+        }),
+    }
+
+    prompts = []
+
+    public async run(): Promise<void> {
+        const { flags } = await this.parse(UpdateOverride)
+        const { headless, project } = flags
+        await this.requireProject(project, headless)
+        const identity = await fetchUserProfile(this.authToken, this.projectKey)
+        if (!identity.dvcUserId) {
+            this.writer.showError('You must set your DevCycle Identity before you can update an override')
+            this.writer.infoMessageWithCommand('To set up your SDK Associated User ID, use', 'dvc identity update')
+            return
+        }
+
+        let { feature, environment, variation } = flags
+        if (headless && (!feature || !environment || !variation)) {
+            this.writer.showError('The feature, environment, and variation flag must be set when using headless mode')
+            return
+        } else if (headless && feature && environment && variation) {
+            const params = await this.populateParametersWithZod(UpdateOverrideDto, this.prompts, {
+                feature, environment, variation
+            })
+            const result = await updateOverride(this.authToken, this.projectKey, feature, params)
+            this.writer.showResults(result)
+            return
+        }
+        if (!feature) {
+            const response = await inquirer.prompt<FeaturePromptResult>([featurePrompt], {
+                token: this.authToken,
+                projectKey: this.projectKey
+            })
+            feature = response.feature._id
+        }
+        if (!environment) {
+            const responses = await inquirer.prompt<EnvironmentPromptResult>(
+                [environmentPrompt],
+                {
+                    token: this.authToken,
+                    projectKey: this.projectKey
+                }
+            )
+            environment = responses.environment._id
+        }
+        if (!variation) {
+            const response = await inquirer.prompt([variationPrompt], {
+                token: this.authToken,
+                projectKey: this.projectKey,
+                featureKey: feature,
+            })
+            variation = response.variation._id
+        }
+
+        const params = await this.populateParametersWithZod(UpdateOverrideDto, this.prompts, {
+            feature, environment, variation
+        })
+
+        const result = await updateOverride(this.authToken, this.projectKey, feature, params)
+        this.writer.successMessage(
+            `Successfully added override for userId ${result.dvcUserId}`)
```

Comment:
can we make the output clearer as to what change was made?

```suggestion
            `Successfully added override for User ID: ${result.dvcUserId} for feature: ${featureKey} on environment: ${environmentKey} to variation: ${variationKey}`)
```

Reply:
I had that before but it was too long so removed it, but I will prettify it

Reply:
you can multi-line split by doing something like:
`Successfully added override for User ID: ${result.dvcUserId}` +
` for feature: ${featureKey} ` etc etc